src/backend/GnuCashBudget.Adapter.Example/Services/AdapterService.cs

```diff
@@ -0,0 +1,18 @@
+using GnuCashBudget.Adapter.Abstractions.Models;
+using GnuCashBudget.Adapter.Example.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace GnuCashBudget.Adapter.Example.Services;
+
+public class AdapterService(IOptions<ExampleOptions> exampleOptions) : IAdapterService
```

Comment:
We probably don't need this class if we just registered IAdapter to DI and used it directly in the controller